### PR TITLE
chore: simplify prettierignore

### PR DESCRIPTION
**Pull request by turadg**
## Description

Removing special cases in prettierignore, came up in https://github.com/Agoric/agoric-sdk/pull/4364#discussion_r799536180

Also removed the nested `.prettierignore` files because they're not read during the root command https://github.com/prettier/prettier/issues/4081

### Security Considerations

--

### Documentation Considerations

Reduces need for documenation

### Testing Considerations

--

### CODE REVIEW

**Comment by mhofman** [comment]
> Removing special cases in prettierignore, came up in [#4364 (comment)](https://github.com/Agoric/agoric-sdk/pull/4364#discussion_r799536180)

What is the plan to keep this file in sync with upstream? Reformatting usually creates headaches.

**Comment by turadg** [comment]
> What is the plan to keep this file in sync with upstream? Reformatting usually creates headaches.

I assume that when someone wants to pull changes from the original source they would pretty it first so they could get a useful diff.

File: .prettierignore [hunk 1]
```diff
@@ -15,10 +15,3 @@ public
 *.json
 *.md
 
-# subrepos
-/packages/xsnap/moddable
-/packages/xsnap/xsnap-native
```
Comment (mhofman) [inline .prettierignore:20]: How do we prevent prettier from reaching into subrepos without this?

Comment (turadg) [reply .prettierignore:20]: Indeed. This was an error introduced when I testing whether nested ignore files worked. (They don't) I'll restore. Thankfully the linter complained too!

**Pull request closed without merging**